{"url":"https://api.github.com/repos/owncloud/core/issues/6135","labels_url":"https://api.github.com/repos/owncloud/core/issues/6135/labels{/name}","comments_url":"https://api.github.com/repos/owncloud/core/issues/6135/comments","events_url":"https://api.github.com/repos/owncloud/core/issues/6135/events","html_url":"https://github.com/owncloud/core/issues/6135","id":23517581,"number":6135,"title":"problems with german umlaute after upgrading php 5.3 -> 5.4","user":{"login":"andyknownasabu","id":377856,"avatar_url":"https://gravatar.com/avatar/6fe5c3be82d8398af232b3f571eced63?d=https%3a%2f%2fidenticons.github.com%2fd8990756e869f6b3199cae2217b676db.png&r=x","gravatar_id":"6fe5c3be82d8398af232b3f571eced63","url":"https://api.github.com/users/andyknownasabu","html_url":"https://github.com/andyknownasabu","followers_url":"https://api.github.com/users/andyknownasabu/followers","following_url":"https://api.github.com/users/andyknownasabu/following{/other_user}","gists_url":"https://api.github.com/users/andyknownasabu/gists{/gist_id}","starred_url":"https://api.github.com/users/andyknownasabu/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/andyknownasabu/subscriptions","organizations_url":"https://api.github.com/users/andyknownasabu/orgs","repos_url":"https://api.github.com/users/andyknownasabu/repos","events_url":"https://api.github.com/users/andyknownasabu/events{/privacy}","received_events_url":"https://api.github.com/users/andyknownasabu/received_events","type":"user","site_admin":false},"labels":[],"state":"closed","assignee":null,"milestone":null,"comments":3,"created_at":"2013-11-30t08:59:11z","updated_at":"2013-11-30t17:59:20z","closed_at":"2013-11-30t17:59:20z","pull_request":{"html_url":null,"diff_url":null,"patch_url":null},"body":"hi,\r\n\r\nsince the latest owncloud version from git now requires php >= 5.4 i upgraded my server. the upgrade went well but i now have problems with file/directory names that contained german umlaute before. they are not shown properly and the sync client throws an error message. what's even worse, the files got removed from my pc  by the sync client.\r\n\r\nseems to be related to\r\nhttp://php.net/manual/de/migration54.incompatible.php#111011\r\ni guess.\r\n\r\nplease fix, thanks.","closed_by":{"login":"kabum","id":245432,"avatar_url":"https://gravatar.com/avatar/a1dea19f88f38ea0b5b5b4d19d5b3107?d=https%3a%2f%2fidenticons.github.com%2f53d8cbb3dbc087a02951dec47438ccdf.png&r=x","gravatar_id":"a1dea19f88f38ea0b5b5b4d19d5b3107","url":"https://api.github.com/users/kabum","html_url":"https://github.com/kabum","followers_url":"https://api.github.com/users/kabum/followers","following_url":"https://api.github.com/users/kabum/following{/other_user}","gists_url":"https://api.github.com/users/kabum/gists{/gist_id}","starred_url":"https://api.github.com/users/kabum/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/kabum/subscriptions","organizations_url":"https://api.github.com/users/kabum/orgs","repos_url":"https://api.github.com/users/kabum/repos","events_url":"https://api.github.com/users/kabum/events{/privacy}","received_events_url":"https://api.github.com/users/kabum/received_events","type":"user","site_admin":false}}